### ISSUE TITLE
Parsing JSON from reanalyze failed.

### ISSUE THREAD

**Issue by Mng12345** [rescript-lang/rescript-vscode#717]
Getting this error with rescript-vscode v1.10.0 on ubuntu22(wsl2), the version of rescript is v10.0.1
```
Parsing JSON from reanalyze failed. The raw, invalid JSON can be reproduced by following the instructions below. Please run that command and report the issue + failing JSON on the extension bug tracker: https://github.com/rescript-lang/rescript-vscode/issues
> To reproduce, run "/home/ming/.vscode-server/extensions/chenglou92.rescript-vscode-1.10.0/server/analysis_binaries/linux/rescript-editor-analysis.exe reanalyze -json" in directory: "/home/ming/projects/mng-gif-maker-v6/res/gif-compress"
```
I reran rescript-editor-analysis.exe in directory `/home/ming/projects/mng-gif-maker-v6/res/gif-compress`, the result json is below:
[reanalyse-log.txt](https://github.com/Mng12345/rescript-extension-debug-files/blob/main/reanalyse-log.txt)

**Comment by cristianoc** [comment]
The attached appears to be valid JSON. Tested with https://jsonlint.com.
See if you can reproduce and get an incorrect one.

**Comment by Mng12345** [comment]
Yes, it's valid json file. The same error message showed every time i save the code.

**Comment by zth** [comment]
Might be a problem with reporting false positives then. When this message keeps appearing, do you still get updated diagnostics from the code analyzer? If you add a new variable at the top level that nothing uses, is that reported right away even if you get that message?

**Comment by Mng12345** [comment]
Yes, the editor works normally, when this message keeps appearing.

**Comment by zth** [comment]
So new issues from the code analyzer is still presented?

**Comment by Mng12345** [comment]
There are just some warnings about unused variables and types. The type tips and auto completion are working normally.
